inventory/src/components/KitIndex/BlueprintSelect.js

```diff
@@ -0,0 +1,196 @@
+      sx={{
+        bgcolor: 'background.paper',
+      }}
+    >
+      <Stack>
+        {kitRecipeData &&
+          kitRecipeData.map((item) => {
+            return (
+              <Accordion
+                data-testid={`blueprint-${item.item_id}`}
+                key={item.item_id}
+                defaultExpanded={true}
+                sx={{ border: '2px solid #000' }}
+              >
+                <AccordionSummary expandIcon={<ExpandMore />}>
+                  <Typography variant='h6'>{item.item_name}</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Stack
+                    data-testid={`blueprint-item-${item.item_id}`}
+                    spacing={2}
+                  >
+                    {item.item_options.map((o) => {
+                      return (
+                        <Stack
+                          data-testid={`blueprint-itemExpiry-${o.item_expiry_id}`}
+                          spacing={1}
+                          key={o.item_expiry_id}
+                        >
+                          <Box
+                            sx={{
+                              display: 'flex',
+                              justifyContent: 'space-between',
+                              alignItems: 'center',
+                            }}
+                          >
+                            <span>ID:</span>
+                            <span>{o.item_expiry_id}</span>
+                          </Box>
+                          <Box
+                            sx={{
+                              display: 'flex',
+                              justifyContent: 'space-between',
+                              alignItems: 'center',
+                            }}
+                          >
+                            <span>Expiry Date:</span>
+                            <span>{o.expiry_date}</span>
+                          </Box>
+                          <Box
+                            sx={{
+                              display: 'flex',
+                              justifyContent: 'space-between',
+                              alignItems: 'center',
+                            }}
+                          >
+                            <span>Quantity:</span>
+                            <TextField
+                              value={getExpiryQuantity(
+                                item.item_id,
+                                o.item_expiry_id,
+                              )}
+                              onChange={(e) =>
+                                kitContentChange(
+                                  o.item_expiry_id,
+                                  item.item_id,
+                                  e.target.value,
+                                  o.quantity,
+                                  item.required_quantity,
+                                )
+                              }
+                              size='small'
```

Comment:
Make the text field mobile responsive, currently text field seems a little too big for mobile